backend/test/test_data/claimEvent.json

```diff
@@ -0,0 +1,11 @@
+{
```

Comment:
We could store this value as a variable inside some javascript file, similar to [this](https://github.com/filecoin-station/spark-evaluate/blob/d2368132e8c8987c87cb5700dc13dfeda291cd17/test/helpers/test-data.js#L11-L17).